_docs/individual-payments.md

```diff
@@ -65,6 +65,14 @@ Unity Bank: 215
 VFD Microfinance Bank: 566
 Wema Bank: 035
 Zenith International: 057
+9 Payment Service Bank Limited: B97
```

Comment:
Could you please ensure the banks are in alphabetical order

Reply:
depending on the ticket that will be merged first, one of us will update this list https://github.com/transferzero/api-documentation/pull/235

Reply:
@thirunjuguna Since the mentioned ticket is already approved, go ahead and merge it first.

Reply:
@thirunjuguna I've updated this with your changes from the mentioned PR.